pkg/addr/svc.go

```diff
@@ -46,6 +48,7 @@ type SVC uint16
 // SVC addresses, use CS_A and DS_A; shorthand versions without
 // the _A suffix (e.g., CS) also return anycast SVC addresses. For multicast,
 // use CS_M, and DS_M.
+// @ decreases
 func ParseSVC(str string) (SVC, error) {
```

Comment:
this should return error.Mem() if the error is not nil

Reply:
Done.